### DISCUSSION TITLE
How to show namespace starting with kubexxx-infra in Kiali dashboard

### DISCUSSION THREAD

**Discussion by widdix123** [kiali/kiali#3723] [Q&A]
I have a personal namespace starting with kube123-infra and i want it be shown in the dashboard ? Currently anything with starting with `kube` is not shown in the dashboard

Any suggestion on this would be helpful ?

**Comment by aljesusg** [comment]
Hi, by default kialiavoid the namespace a that start by kube (Note: there are namespaces for kubernetes management that start by kube)

In the CR file you can add the namespace to avoid this

```yaml

namespaces:
#      exclude:
#      - "istio-operator"
#      - "kube.*"
#      - "openshift.*"
#      - "ibm.*"
#      - "kiali-operator"

```

**Reply by jmazzitelli** [reply]
here's a link for completeness: https://github.com/kiali/kiali-operator/blob/v1.30.0/deploy/kiali/kiali_cr.yaml#L89-L118
